src/PythonMigrationViewExtension/GraphPythonDependencies.cs

```diff
@@ -14,9 +14,9 @@ public class GraphPythonDependencies
         private ViewLoadedParams ViewLoaded { get; set; }
 
         // A dictionary to mark Custom Nodes if they have a IronPython dependency or not. 
-        private static Dictionary<Guid, CNPythonDependency> CustomNodePythonDependency = new Dictionary<Guid, CNPythonDependency>();
+        internal static Dictionary<Guid, CNPythonDependency> CustomNodePythonDependency = new Dictionary<Guid, CNPythonDependency>();
```

Comment:
is this internal for testing or so it can be called from some other assembly?

Reply:
because this an internal field I guess it should be camelCase.

Reply:
Internal because it is being used by other files.